.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,19 @@
-<!-- describe the changes you have made here: what, why, ... 
-     Link issues by using the following pattern: [#333](https://github.com/JabRef/jabref/issues/333) or [koppor#49](https://github.com/koppor/jabref/issues/47).
-     The title of the PR must not reference an issue, because GitHub does not support autolinking there. -->
+<!-- 
+Describe the changes you have made here: what, why, ... 
+Link issues by using the following pattern: #333.
+If you fixed a koppor issue, link it with following pattern: [koppor#47](https://github.com/koppor/jabref/issues/47).
+The title of the PR must not reference an issue, because GitHub does not support autolinking there.
+-->
 
+## Checklist
```

Comment:
The body of the PR is always the most important one, thus I don't like the heading here as it puts too much emphasize on something relatively unimportant.

Reply:
I wanted the thing to be more intuitively separated. Using `--` it always felt a bit off. I wondered whether we should add a section `## Description`